### PR TITLE
ToggleShieldsPacket vs ToggleRedAlertPacket

### LINKED ISSUE #181: ToggleShieldsPacket link points to ToggleRedAlertPacket
In the packet types table, clicking the link saying ToggleShieldsPacket shows the ToggleRedAlertPacket packet structure. Also, the section for ToggleShieldsPacket has the id for ToggleRedAlertPacket:
`<section id="toggleredalertpacket">`

### PR DESCRIPTION
**Pull request by NoseyNick**
Easy fix for #181 - mislabelled section

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*~
```
Comment (JordanLongstaff) [inline .gitignore:1]: What is this for?

**Review by JordanLongstaff — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (NoseyNick) [reply .gitignore:1]: Oh, my editor, when I edit `index.html`, keeps a backup as `index.html~`
It's fairly normal to have a more detailed `.gitignore` with a bunch of things like `*.bak`, `*.o`, `OLD/*`, `*.cert`, `*.tmp` or other stuff that you **DON'T** want to include in `git add` / `git commit` / etc. Most git projects I work on have a **much** more detailed `.gitignore` but I thought it was important to at least get one that can ignore my own editor files. Feel free to send a PR for more if your editor uses something else, or if there's any other rogue files you think we should ignore going forward.

Comment (NoseyNick) [reply .gitignore:1]: I'll admit it's not relevant to #181 and should arguably have been in a different PR, but... Meh!   :man_shrugging:

**Review by StarryWisdom — APPROVED** [review]
assuming that the Packet Types table is autogenerated from the same edit it looks correct

**Comment by NoseyNick** [comment]
Re-pushed with signature for that `Verified` thing up there :arrow_upper_right:

**Comment by NoseyNick** [comment]
> assuming that the Packet Types table is autogenerated

Seems to be. Worked on my local copy much better than the live one anyway.

**Comment by NoseyNick** [comment]
Dropping in favour of #188 which not only fixes the duplicate `<section id="toggleredalertpacket">` but also a big bunch of other HTML fixes.
Will do the `.gitignore` another time... Or just keep it my side by including `.gitignore` **in** `.gitignore`

**Pull request closed without merging**